pkg/cfaws/profiles.go

```diff
@@ -152,10 +160,10 @@ func (c CFSharedConfigs) ProfileNames() []string {
 	return names
 }
 
-func (c *CFSharedConfig) AssumeConsole(ctx context.Context, args []string) (aws.Credentials, error) {
-	return AssumerFromType(c.ProfileType).AssumeConsole(ctx, c, args)
+func (c *CFSharedConfig) AssumeConsole(ctx context.Context, browserOpts browsers.BrowserOpts, configOpts ConfigOpts) (aws.Credentials, error) {
```

Comment:
I think we should remove the browser opts param here and on all the assumer interfaces.

The assumers are intended to generate credentials.
The Browser package uses those credentials to get a browser session .


So they have seperate concerns and there should be no options in browserOpts which impact the the assumers